uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -296,6 +298,52 @@ public function testGetReturnsOnlyPublicHolidayLeaveRequestsIfThePublicHolidayIs
     $this->assertEquals($publicHoliday->date, $result['values'][0]['from_date']);
   }
 
+  public function testGetIncludesToilLeaveRequests() {
+    HRJobContractFabricator::fabricate(['contact_id' => 1], ['period_start_date' => '-1 day']);
+    $leaveRequestStatuses = array_flip(LeaveRequest::buildOptions('status_id'));
+
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'must_take_public_holiday_as_leave' => true,
+      'allow_accruals_request' => true,
+    ]);
+
+    $leaveRequest1 = LeaveRequestFabricator::fabricateWithoutValidation([
+      'contact_id' => 1,
+      'type_id' => $absenceType->id,
+      'from_date' => CRM_Utils_Date::processDate('+1 day'),
+      'to_date' => CRM_Utils_Date::processDate('+2 days'),
+      'status_id' => $leaveRequestStatuses['Approved']
+    ], true);
+
+    $leaveRequest2 = LeaveRequestFabricator::fabricateWithoutValidation([
+      'contact_id' => 1,
+      'type_id' => $absenceType->id,
+      'from_date' => CRM_Utils_Date::processDate('+1 day'),
+      'to_date' => CRM_Utils_Date::processDate('+2 days'),
+      'status_id' => $leaveRequestStatuses['Waiting Approval']
+    ], true);
+
+    $expiryDate = new DateTime('+100 days');
+    $toilRequest = TOILRequest::create([
+      'type_id' => $absenceType->id,
+      'contact_id' => 1,
+      'status_id' => 1,
+      'from_date' => date('YmdHis'),
```

Comment:
Please use CRM_Utils_Date::processDate() to keep consistency with the calls to the LeaveRequest fabricator